cpp/ql/src/Critical/FlowAfterFree.qll

```diff
@@ -101,7 +101,7 @@ predicate isFree(DataFlow::Node n, Expr e, DeallocationExpr dealloc) {
   exists(Expr conv |
     e = conv.getUnconverted() and
     conv = dealloc.getFreedExpr().getFullyConverted() and
-    conv = n.asConvertedExpr()
+    conv = n.(DataFlow::PostUpdateNode).getPreUpdateNode().asConvertedExpr()
```

Comment:
I see that we're (usually) interested in the post-update node for the purposes of "flow after free".  But I think the QLDoc for this function needs updating.

Reply:
Yeah, both `cpp/use-after-free` and `cpp/double-free` now select the post-update node of the argument for their dataflow tasks.

I've updated the QLDoc in f284fde93cc7f986e12bfc3dbc587124c789ab71.

Reply:
... oops. Fixed in 97f2be9b82ff14398a4af0fef6a51078901c8d90